Project/Source/Components/ComponentCamera.cpp

```diff
@@ -28,7 +28,8 @@ void ComponentCamera::Init() {
 }
 
 void ComponentCamera::Update() {
-	UpdateFrustum();
+	//Active camera is the only one that updates frustum, if additional cameras were to require an update before, a different system would be required
+	//This update is handled in ModuleCamera.cpp
```

Comment:
I think an update on all cameras might be necessary so that we can draw the camera gizmos.

Reply:
Do you mean an update on all cameras from ModuleCamera or keeping the ComponentCamera::Update and the ModuelCamera updating the active frustum just in case?